src/test/test-app.ts

```diff
@@ -60,5 +61,6 @@ export async function bootstrapTestApp(): Promise<INestApplication> {
 
   const app = module.createNestApplication();
   app.use(json({ limit: '10mb' }));
+  app.use(cookieParser('secret'));
```

Comment:
Should we use the config value here too?

Reply:
updated